service.unmanic/default.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
```

Comment:
SPDX

Reply:
What part of the SPDX are you after?
https://spdx.org/licenses/GPL-3.0-only.html
I added the last one "Standard License Header"

Are you just wanting the heater to be:
```
# -*- coding: utf-8 -*-
# GPL-3.0-only
```

Reply:
Yes, it should be:

```# SPDX-License-Identifier: GPL-3.0-only```

And I don't think you need `# -*- coding: utf-8 -*-` if you are not using any UTF-8 chars in the file.